packages/devtools_app/lib/src/charts/flutter/flame_chart.dart

```diff
@@ -775,40 +809,54 @@ class FlameChartNode<T> {
 
 mixin FlameChartColorMixin {
   int _uiColorOffset = 0;
-  Color nextUiColor() {
+  Color nextUiColor({bool resetOffset = false}) {
+    if (resetOffset) _uiColorOffset = 0;
```

Comment:
add curly braces around if statements that don't end with a return.

Reply:
done